packages/lesswrong/components/messaging/ConversationItem2.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { registerComponent, Components } from "../../lib/vulcan-lib";
+import * as _ from "underscore";
+import { truncate } from "../../lib/editor/ellipsize";
+import { useClickableCell } from "../common/useClickableCell";
+import classNames from "classnames";
+import { conversationGetTitle2 } from "../../lib/collections/conversations/helpers";
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    backgroundColor: theme.palette.background.pageActiveAreaBackground,
+    display: "flex",
+    alignItems: "center",
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    padding: 16,
+    borderBottom: theme.palette.border.grey200,
+    cursor: "pointer",
+    "&:hover": {
+      background: theme.palette.grey[50],
+    },
+  },
+  rootSelected: {
+    background: theme.palette.grey[100],
+  },
+  profileImage: {
+    marginRight: 12,
+  },
+  content: {
+    width: "100%"
+  },
+  titleRow: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  title: {
+    overflow: "hidden",
+    display: "-webkit-box",
+    "-webkit-box-orient": "vertical",
+    "-webkit-line-clamp": 1,
+    color: theme.palette.grey[1000],
+    fontSize: 16,
+    fontWeight: 700,
+    lineHeight: '21px'
+  },
+  date: {
+    color: theme.palette.grey[600],
+    fontSize: 14,
+    fontWeight: 500,
+    lineHeight: '21px',
+    marginLeft: 4,
+    marginRight: 4,
+  },
+  preview: {
+    overflow: "hidden",
+    display: "-webkit-box",
+    "-webkit-box-orient": "vertical",
+    "-webkit-line-clamp": 1,
```

Comment:
`webkit-line-clamp` can be buggy on safari. If we're clamping to a single line it's probably safer to just do it the old-fashioned way:
```
  whiteSpace: "nowrap",
  textOverflow: "ellipsis",
```
I think you'll also need to remove the `width: "100%"` on `.content` and replace it with `minWidth: 0`.

(The same goes for everywhere we have `"-webkit-line-clamp": 1`).